README.md

```diff
@@ -185,6 +185,29 @@ async def purge(ctx: commands.Context, tz_info: str, military_time: bool):
 ```
 </details>
 
+<details><summary><b>Asset handler</b></summary>
+In order to prevent the transcripts from being broken either when a channel is deleted or by the newly introduced 
+restrictions to media links in discord, chat-exporter now supports an asset handler. `chat_exporter.construct.
+asset_handler.AssetHandler` servers as a template for you to implement your own asset handler. As example we provide 
+to basic versions of an asset handler, one that stores the assets locally and one that uploads them to a discord. Of 
+course the second one is also broken, but it should give a good idea on how to implement such an `AssetHandler`.
+If you dont specify an asset handler, chat-exporter will use the normal (proxy) urls for the assets.
+The important part of your implementation is, that you have to overwrite the url and proxy_url attribute of the 
+Attachment in your implementation of `AssetHandler`. The url attribute should be the url where the asset is available.
+
+
+**Example:**
+```python
+import io
+
+...
+
+from chat_exporter.construct.asset_handler import LocalFileHostHandler
+file_handler = LocalFileHostHandler(base_path="/usr/share/assets",
+                                    url_base="https://example.com/assets")
+transcript = await chat_exporter.export(channel, guild=channel.guild, asset_handler=file_handler)
+```
```

Comment:
Could you update the example adding in formatting and I make it more 'real' (people will copy and paste exactly what is there and complain things are broken).